### ISSUE TITLE
[T9][Accessibility][Functional]:Unable to make selection in familly tab -Aotu_immune_diseases - Expand and collapse

### ISSUE THREAD

**Issue by SR027213** [SR027213/Feb-19-Bug-Bash#34]
steps to reproduce:
Unable to make selection in familly tab -Aotu_immune_diseases - Expand and collapse
